### ISSUE TITLE
Module Failure

### ISSUE THREAD

**Issue by josekampos** [CheckPointSW/cpAnsible#1]
Nice module, i just have to figure out how it works... i was doing some testing, but got stuck in this error:

TASK [login] *******************************************************************************************************************************************
fatal: [127.0.0.1]: FAILED! => {"changed": false, "module_stderr": "Shared connection to 127.0.0.1 closed.\r\n", "module_stdout": "Traceback (most recent call last):\r\n  File \"/tmp/ansible_OUVfcL/ansible_module_check_point_mgmt.py\", line 8, in <module>\r\n    from cp_mgmt_api_python_sdk.lib import APIClient, APIClientArgs\r\nImportError: No module named cp_mgmt_api_python_sdk.lib\r\n", "msg": "MODULE FAILURE", "rc": 0}

Any advice on how to fix this?

**Comment by JonTheNiceGuy** [comment]
When you got the module, you didn't pick up the library that goes with it...

Did you git clone or just download the module?

**Comment by josekampos** [comment]
I did a git clone, and followed the instructions (i think):

ansible.cfg:
library        = /usr/share/ansible/

automator@vm-ansible:/usr/share/ansible/check_point_mgmt$ ls -la
total 20
drwxr-xr-x 2 root root  4096 Nov 30 17:24 .
drwxr-xr-x 3 root root  4096 Nov 30 17:23 ..
-rw-r--r-- 1 root root 10002 Nov 30 17:20 check_point_mgmt.py

and 

automator@vm-ansible:/usr/lib/python2.7/site-packages$ ls -la
total 28
drwxr-xr-x  3 root root  4096 Nov 30 10:17 .
drwxr-xr-x 27 root root 20480 Nov 30 10:16 ..
drwxr-xr-x  7 root root  4096 Nov 30 10:17 cp_mgmt_api_python_sdk

I'm not an expert at ansible module loading, i'm i missing something?

**Comment by JonTheNiceGuy** [comment]
Ah, right I see what you mean. OK, I did things a bit differently, [I merged the API interface (cp_mgmt_api_python_sdk) and the check_point_mgmt.py into a single file](https://gist.github.com/JonTheNiceGuy/270322a511122ab05525d37e22e3e562) which I place in the "library" folder where my playbooks are.

Note, this hasn't been updated with the latest version of either the module, or the SDK, but still should work!

I'll let someone else also take a look at this to see if you've missed anything from doing it "the Right Way" for Check Point (I'm a 3rd party who also consumes this!)
